public/app/features/dashboard-scene/settings/version-history/utils.ts

```diff
@@ -0,0 +1,100 @@
+import { compare, Operation } from 'fast-json-patch';
+// @ts-ignore
```

Comment:
Why this `@ts-ignore`? Is there any reason?

Reply:
No idea why that is there, good eye, will investigate!

Reply:
Looks like the ignore is there because it needs a definitions file. There's an open PR in the package repo for it but it's hanging since 2021 and the last release of this package was 2019. Might need to look into a replacement for this one, potentially?